### DISCUSSION TITLE
[Feature Request] Custom Geojson Layers

### DISCUSSION THREAD

**Discussion by mertbakir** [grafana/grafana#48080] [Dashboards, Panels & Visualizations]
Hi, 

I couldn't find a way to add more geojsons here, geomap panel. I guess there is no way atm
![image](https://user-images.githubusercontent.com/43188411/164530753-ec5ca3ef-6019-4c6f-8533-b326f27e66f7.png)

I think it would be really useful if we could just get a geojson from database by simplying querying. 

- as a binary file from db.
- jsonb format from db.
- or geometry type with postgis support.

**Comment by sesoko838** [comment]
Definitely find this aspect of the GeoJSON difficult.  Currently using this solution here but would be a big plus if they could make this easier https://community.grafana.com/t/adding-custom-geojson-to-geomap/57991/3

**Comment by mertbakir** [comment]
@sesoko838 I discovered a custom plugin yesterday, you may find it useful. It supports postgis.
https://github.com/orchestracities/map-panel

**Reply by yosiasz** [reply]
This plugin solved my problem, thanks @mertbakir

**Comment by yosiasz** [comment]
Check this out in case it helps

https://github.com/grafana/worldmap-panel/issues/6#issuecomment-1159118634

But cant seem to find a way to make that url dynamic even via a dashboard variable

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
